SpatialGDK/Extras/schema/net_owning_client_worker.schema

```diff
@@ -12,4 +12,20 @@ component NetOwningClientWorker {
     // EntityACL entry for the client RPC endpoint (and Heartbeat component, 
     // if present).
     option<string> worker_id = 1;
```

Comment:
If USLB is always on, maybe we don't need to store client worker ID here? not sure